specs/refactor_gluster_integration_get_state_dump.adoc

```diff
@@ -148,6 +148,9 @@ Last allocated port: 49153
 
 * General refactoring of code handling the gluster get-state output and add
 all missing volume attributes to the Volume object
+** Additional fields available under volume details from get-state output
+like transport_type, snap_count, stripe_count, replica_count, arbiter_count,
+quorum_status, ... to be added to central store
 
 * Modify the tendrl definitions specifications for gluster and add all the
 missing attributes to Volume object
```

Comment:
- list down all attributes you are planning to add
- structure json which will updated to etcd
- Any changes to tendrl definitions to be written down here(snip from tendrl definitions)